### ISSUE TITLE
Accessibility errors found in your template

### ISSUE THREAD

**Issue by wapyce** [woodcoding/woodcoding.github.io#1]
The user @carlsonsantana validate your site template "https://woodcoding.github.io" and found these accessibility errors:
* Page https://woodcoding.github.io/:
  * Issue *1eb8c1ef-ca7b-45f0-aeba-b8d31e28daa7*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.01:1. Recommendation: change text colour to #d74000.
    * **Selector**: `#sidebar > div > section > div:nth-child(3) > a`
    * **Context**: `<a href="/atom.xml" rel="alternate">
              <i class="fa fa-...</a>`
  * Issue *8fe8a2c0-f699-4b75-afa7-9e30ae7d9a9f*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.85:1. Recommendation: change text colour to #767676.
    * **Selector**: `#sidebar > div > section > nav > div:nth-child(3) > a > span:nth-child(2)`
    * **Context**: `<span class="site-state-item-name">标签</span>`
  * Issue *ab312aa7-6063-4ad8-8f8d-47752c0b6e79*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.85:1. Recommendation: change text colour to #767676.
    * **Selector**: `#sidebar > div > section > nav > div:nth-child(2) > a > span:nth-child(2)`
    * **Context**: `<span class="site-state-item-name">分类</span>`
  * Issue *9105e086-be7c-4bcc-b23f-3929b063cec3*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.85:1. Recommendation: change text colour to #767676.
    * **Selector**: `#sidebar > div > section > nav > div:nth-child(1) > a > span:nth-child(2)`
    * **Context**: `<span class="site-state-item-name">日志</span>`
  * Issue *a14def14-9f5c-49e8-a718-ecb0a19e9d40*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.85:1. Recommendation: change text colour to #767676.
    * **Selector**: `#posts > article > header > div > div > span:nth-child(3)`
    * **Context**: `<span title="阅读时长">
                  0
          ...</span>`
  * Issue *c5b1f2ba-d9ab-4fc3-832e-381443040453*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.85:1. Recommendation: change text colour to #767676.
    * **Selector**: `#posts > article > header > div > div > span:nth-child(2)`
    * **Context**: `<span class="post-meta-item-text">阅读时长</span>`
  * Issue *1e9500c8-b85d-4773-bbff-e8221fd87027*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.85:1. Recommendation: change text colour to #767676.
    * **Selector**: `#/%E6%9D%82%E9%A1%B9%E6%9D%82%E8%B0%88/2018/02/01/Hello-world/ > span:nth-child(3)`
    * **Context**: `<span class="post-meta-item-text">阅读次数 </span>`
  * Issue *fbe71fac-c8d7-40f7-a82e-2b7e90edb289*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.85:1. Recommendation: change text colour to #767676.
    * **Selector**: `#/%E6%9D%82%E9%A1%B9%E6%9D%82%E8%B0%88/2018/02/01/Hello-world/ > span:nth-child(1)`
    * **Context**: `<span class="post-meta-divider">|</span>`
  * Issue *de49ce00-354e-486b-8666-9b8cae6f0a16*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.85:1. Recommendation: change text colour to #767676.
    * **Selector**: `#posts > article > header > div > span:nth-child(2) > span:nth-child(3)`
    * **Context**: `<span class="post-meta-item-text">分类于</span>`
  * Issue *b9393dbf-5f94-4351-af86-5364e25dfba5*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.85:1. Recommendation: change text colour to #767676.
    * **Selector**: `#posts > article > header > div > span:nth-child(2) > span:nth-child(1)`
    * **Context**: `<span class="post-meta-divider">|</span>`
  * Issue *45e0b16d-736a-49a4-b2a5-ece47ab9ea62*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.85:1. Recommendation: change text colour to #767676.
    * **Selector**: `#posts > article > header > div > span:nth-child(1) > time`
    * **Context**: `<time title="创建于" itemprop="dateCreated datePublished" datetime="2018-02-01T22:00:00+00:00">
                2018-02-01
   ...</time>`
  * Issue *bfa90e9a-7837-432c-b077-60fbb8f7810c*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.85:1. Recommendation: change text colour to #767676.
    * **Selector**: `#posts > article > header > div > span:nth-child(1) > span:nth-child(2)`
    * **Context**: `<span class="post-meta-item-text">发表于</span>`
  * Issue *1e13d2c1-e6d5-44cc-aca6-4608f98665c2*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_3.1_3_1.F68
    * **Message**: This form field should be labelled in some way. Use the label element (either with a "for" attribute or wrapped around the form field), or "title", "aria-label" or "aria-labelledby" attributes as appropriate.
    * **Selector**: `#local-search-input`
    * **Context**: `<input autocomplete="off" placeholder="搜索..." spellcheck="false" type="text" id="local-search-input">`
  * Issue *f41ae470-a606-4765-9a3a-dcb6bb4c5ddc*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.InputText.Name
    * **Message**: This textinput element does not have a name available to an accessibility API. Valid names are: label element, title undefined, aria-label undefined, aria-labelledby undefined.
    * **Selector**: `#local-search-input`
    * **Context**: `<input autocomplete="off" placeholder="搜索..." spellcheck="false" type="text" id="local-search-input">`
  * Issue *1095f9d7-ca7b-49c7-9e91-3286ada00953*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Button.Name
    * **Message**: This button element does not have a name available to an accessibility API. Valid names are: title undefined, element content, aria-label undefined, aria-labelledby undefined.
    * **Selector**: `#header > div > div > div:nth-child(2) > button`
    * **Context**: `<button>
      <span class="btn-bar"></...</button>`

You can check these accessibility errors using [pa11y](https://github.com/pa11y/pa11y).
You can view the [full validation results in our website](https://wapyce.herokuapp.com/validation/validations/9c104d36-f3df-44d2-ac33-d7020b215614/).
